packages/react-components/review/react-components.api.md

```diff
@@ -635,6 +635,8 @@ export type MessageThreadProps = {
     onLoadPreviousChatMessages?: (messagesToLoad: number) => Promise<boolean>;
     onRenderMessage?: (messageProps: MessageProps, messageRenderer?: MessageRenderer) => JSX.Element;
     onRenderFileDownloads?: (userId: string, message: ChatMessage) => JSX.Element;
+    fileDownloadErrorMessage?: string;
```

Comment:
This API seems to be scoped too tight -- If we want to add a way for `MessageThread` to show an error, it should be agnostic of the error being shown (i.e., `errorBarMessage` and `onDismissErrorBar`).

Even then, this feature doesn't seem to fit a `MessageThread`. Is there any reason that the `ChatComposite` can't show a `MessageBar` on top of the `MessageThread` (just like we show varius banners for call recording, `ErrorBar` for errors from ACS services etc.)